roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -14,6 +14,22 @@
     l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l_crio_registries)) }}"
   when: l2_docker_additional_registries
 
+- set_fact:
+    l_openshift_image_tag: "{{ openshift_image_tag | string }}"
+  when: openshift_image_tag is defined
+
+- set_fact:
+    l_openshift_image_tag: "latest"
+  when:
+    - openshift_image_tag is not defined
+    - openshift_release == "latest"
+
+- set_fact:
+    l_openshift_image_tag: "v{{ openshift_release | string }}"
```

Comment:
@giuseppe according to https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L38, openshift_release would be already set with 'v' prefix, e.g: v3.7, so this line should NOT add one more 'v' prefix.